data/sheragi outfits.txt

```diff
@@ -0,0 +1,220 @@
+# Copyright (c) 2020 by Fernando S. Sáez
+#
+# Endless Sky is free software: you can redistribute it and/or modify it under the
+# terms of the GNU General Public License as published by the Free Software
+# Foundation, either version 3 of the License, or (at your option) any later version.
+#
+# Endless Sky is distributed in the hope that it will be useful, but WITHOUT ANY
+# WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+# PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+# Outfits
+
+outfit "Electromagnetic Battery"
+	category "Power"
+	cost 1000000
+	thumbnail "outfit/embattery"
+	"mass" 150
+	"outfit space" -150
+	"shield generation" 0.8
+	"shield energy" 0.8
+	"shield heat" 0.8
+	"energy capacity" 20000
+	description `A massive superconductive coil, capable of storing large amounts of energy in a magnetic field, and capable of discharging all of it in a fraction of a second. The magnetic field also provides a small boost to the shield system of the ship. Sheragi shield systems seems very underdeveloped otherwise.`
+
+outfit "Fusion Drive"
+	category "Engines"
+	cost 80000000
+	thumbnail "outfit/fusiondrive"
+	"mass" 480
+	"outfit space" -480
+	"engine capacity" -230
+	"special slot" 2
```

Comment:
`special slot` sounds very generic. If the goal of this slot is to tie the drive to the ship, then I suggest using a more specific name. I would also expect a "special slot" -1 or -2 somewhere else to make this do something.

It is possible to put all engine stats directly on a ship (effectively making the engine internal to the ship).

Reply:
Whoops, that is from way back when i had a different idea for the fusion drive's workings.  I'm going to remove that line.